README.md

```diff
@@ -176,6 +181,20 @@ yarn start
 **Note**: in a development environment where a captive portal may not be available,
 you can use the default sample captive portal login and logout URLs.
 
+#### Removing sections of configuration
+
+To remove the specific section of the configuration, you can use the `null` keyword to delete
+that field or section during the building process. For example, to remove social login links, use
+
+```yaml
+login_form:
+  social_login:
+    links: null
```

Comment:
are you sure this section is still relevant after the latest changes?

Reply:
No, I am changing it.